riscv-asm.md

```diff
@@ -183,6 +183,40 @@ Directive    | Arguments                      | Description
 .balign      | b,[pad_val=0]                  | byte align
 .zero        | integer                        | zero bytes
 .variant_cc  | symbol_name                    | annotate the symbol with variant calling convention
+.attribute   | name, value                    | RISC-V object attributes, more detailed description see [.attribute](#.attribute).
+
+## <a name=.attribute></a> `.attribute`
+
+`.attribute` directive is used for record information about an object
+file/binary that a linker or runtime loader needs to check compatibility.
+
+More information can refer [attribute section in RISC-V psABI](https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-elf.adoc#attributes).
+
+`.attribute` take two argument, first argument of `.attribute` is the symbolic
+name of attribute or the attribute number, the prefix `Tag_RISCV_` can be
+omitted, and second argument can be string or number.
```

Comment:
So is `attribute stack_align, "16"` legal? (currently only `attribute stack_align, 16` is accepted in LLVM).

Reply:
Oh, good catch, the description too loosely here,  `.attribute stack_align, 16` is the only valid input here, and it will check the attribute number, string value if the tag number is odd and an integer value if the tag number is even.